.golangci.yml

```diff
@@ -1,79 +1,124 @@
-# Created based on v1.51.0
-# NOTE: Keep this in sync with the version in .github/workflows/linter.yml
+# v1.2.0. Created based on golangci-lint v1.56.2
 
 run:
-  modules-download-mode: readonly
+  timeout: 5m
   skip-dirs-use-default: false
+  modules-download-mode: readonly
+  allow-serial-runners: true
   skip-dirs:
-    - internal
+    - internal # TODO: Do not ignore interal
 
 output:
   sort-results: true
+  uniq-by-line: false
 
 linters-settings:
-  testifylint:
-    enable-all: true
-    disable:
-      - go-require
+  depguard:
+    rules:
+      all:
+        list-mode: lax
+        deny:
+          - pkg: "flag"
+            desc: '`flag` package is only allowed in main.go'
+          - pkg: "io/ioutil"
+            desc: '`io/ioutil` package is deprecated, use the `io` and `os` package instead'
+          # TODO: Prevent using these without a reason
+          # - pkg: "reflect"
+          #   desc: '`reflect` package is dangerous to use'
+          # - pkg: "unsafe"
+          #   desc: '`unsafe` package is dangerous to use'
```

Comment:
Each use should be justified with an according `//nolint:depguard // Explain why we use the "unsafe" package here`.

Reply:
> Each use should be justified with an according `//nolint:depguard // Explain why we use the "unsafe" package here`.

We use both oftently. I think it can be disabled